component/alert.css

```diff
@@ -0,0 +1,77 @@
+#alert-component{
+    height: fit-content;
+    display: flex;
+    flex-direction: column;
+}
+
+.alert{
+    margin: 2rem;
+    height: fit-content;
+}
+
+/* Common for all alert component  */
+#alert-component .alert-type{ 
+    height: fit-content;
+    flex-direction: column;
+    display: flex;
+    flex-wrap: wrap;
+    background-color: white;
+}
+
+.alert{
+    padding: 1.5rem;
+    border-radius: 1rem;
+}
+
+/* Simple Alert  */
+.alert-primary{
+    background-color: #cce5ff;
+    color: 	#0275d8;
+}
+
+.alert-secondary{
+    background-color: #e2e3e5;
+    color: #576574;
+}
+
+.alert-success{
```

Comment:
Add variable naming to all colours , everything else looks good

Reply:
ok

Reply:
variables added for color